### PR TITLE
submitting my link

### PR DESCRIPTION
**Pull request by sumandroid**

### CODE REVIEW

**Comment by sumandroid** [comment]
I am submitting my tic tac toe assignment link

File: ruby_programming/testing_with_rspec/project_testing_your_ruby_code.md [hunk 1]
```diff
@@ -100,7 +100,6 @@ Only write exactly enough code to make your test pass.  Oftentimes, you'll end u
 Send us your solution so we can show others! Submit a link to the Github repo with your files in it here using any of the methods listed on the [contributing page](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md).  Please include your partner's github handle somewhere in the description if they would like attribution.
 
 * Add your solution below this line!
-* [Javier Machin's solution](https://github.com/Javier-Machin/Connect_Four)
```
Comment (JonathanYiv) [inline ruby_programming/testing_with_rspec/project_testing_your_ruby_code.md]: Hmm, it looks like you're deleting another student's solution. Could you add this back in?

File: web_development_101/the_backend/ruby_project.md [hunk 2]
```diff
@@ -28,7 +28,6 @@ Submit a link below to this [file](https://github.com/TheOdinProject/ruby_course
 * ["Official" Test-First Ruby solutions](https://github.com/ultrasaurus/test-first-teaching/tree/master/learn_ruby) (look for the "solutions" folder inside each exercise)
 
 * Add your solution below this line!
-* [Austin Fisher's solution](https://github.com/austin2016/learn_ruby)
```
Comment (JonathanYiv) [inline web_development_101/the_backend/ruby_project.md]: Hmm, it looks like you're deleting another student's solution. Could you add this back in?

**Comment by sumandroid** [comment]
Will update it

**Comment by sumandroid** [comment]
Hey Jonathan I updated the pull request please verify

**Pull request merged**